client/components/chart/bar.jsx

```diff
@@ -137,26 +137,22 @@ module.exports = React.createClass( {
 	render: function() {
 		debug( 'Rendering bar', this.state );
 
-		var barStyle,
-			barClass,
-			count = this.props.count || 1;
-
-		barClass = { chart__bar: true };
+		const barClass = { chart__bar: true };
```

Comment:
Nevermind my prior comment below, but perhaps this could still be cleaned up a bit

```
const barClass = classNames( 'chart__bar', this.props.className )
```